README.md

```diff
@@ -1,17 +1,21 @@
 # Example repository for using GAP on Binder
 
-[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/gap-system/gap-docker-binder/master)
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/gap-system/gap-docker/master)
 
 This repository should show you how to use deploy [GAP](https://wwww.gap-system.org) Jupyter notebooks to Binder.
+If you simply want to try out GAP on mybinder.org, you can click the launch badge above.
 
-## Using GAP on Binder
+## Using existing jupyter notebooks
+
+GAP provides several Binder-ready Docker images, for example:
```

Comment:
The images are Binder-ready, in a sense that another repository could pull them in its Docker file, and run them on Binder - not in a sense that the repositories where those files are created could be run on Binder. I started to review your https://github.com/gap-system/gap-docker-stable-4.11/pull/4 and https://github.com/gap-system/gap-docker-master/pull/9 and figured out what will go wrong if you try to do so.